### ISSUE TITLE
Fetch is no working since r295777

### ISSUE THREAD

**Issue by dpino** [clopez/webkit-testhunter#11]
Revision r295777 was the last revision commited into SVN ([6c074d1e52c555d43db5738caf6007c988b85ff3](https://github.com/WebKit/WebKit/commit/6c074d1e52c555d43db5738caf6007c988b85ff3)). Since then, the script `fetch.sh` is failing as it tries to parse a SVN revision id.

```
$ ./fetch gtk-release
Fetching results for bot: GTK-Linux-64-bit-Release-Tests
...............
r295768... :
r295769... :
r295770... :
r295772... :
r295774... :
r295776... :
r295777... :
rNone... Fatal: Revision should be numeric and I got revision "None" for http://build.webkit.org/results/GTK-Linux-64-bit-Release-Tests/251785%40main%20%288057%29/full_results.json
```

Worst of all, it seems there's no equivalent identifier in the fetched JSON results.

**Comment by clopez** [comment]
The json files uploaded by the bots have a null value on the revision field.

I have opened this upstream in order to fix it: https://bugs.webkit.org/show_bug.cgi?id=244960

Once this lands upstream, we have to do several changes here to fix this issue:

1) Download the json files from the server that have the null value as revision and update the field (we can guess the revision number from the URL)
2) For consistency reasons is a good idea also to do a massive rename of the current already download json results to update them from the old svn-rev-numbers to the new style of IDs of `number@main`.
3) Adapt the program to work with the new IDs instead of SVN revision numbers
4) Remove from the program the initial check of the SVN server to guess the initial revision adding the test, since SVN not longer works

**Comment by clopez** [comment]
I have updated all the json files with the new identifiers and also updated the code to work with the new identifiers.
This are the commits merged into master:

02a6201b05 Update code to work with the new Git identifiers
8639ae757c Update results from fetch
d47bf5efc7 Update fetch to only accept the new Git format for identifiers
18dd008886 Remove helper scripts used for the mass-renaming and update to Git identifiers
0e9689f779 Rename results to the new Git identifiers (6 and last)
066aef5742 Update revision field in results to the new Git identifiers (5)
144097a494 Rename results to the new Git identifiers (4)
0c2e20ff45 Remove results from SVN revisions that don't have a Git identifier (3)
461deb91eb Update missed key in svn-to-id dict
c2321f57b9 Update revision field in results to the new Git identifiers (2)
d3bf1ba7db Rename results to the new Git identifiers (1)
1bc7befa68 rename script: stage in two steps the rename and the update
b1502e5f04 Update jsonresults again before renaming
13bde9b6e7 Add helper scripts used for the mass-rename and fixes from svn to git
d31be33c2b Update the jsonresults after fixing fetch
e4933ac036 fetch: Allow to download results after migration to GIT.

Note: git pull will take a while the first time because all of the json results were overwritten (and there are a lot of them)

**Comment by clopez** [comment]
I'm closing this, please re-open if something doesn't work.
